toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -163,3 +163,6 @@ p11-kit-trust-0.23.16.1-2.cm1.aarch64.rpm
 ca-certificates-shared-20200720-8.cm1.noarch.rpm
 ca-certificates-tools-20200720-8.cm1.noarch.rpm
 ca-certificates-base-20200720-8.cm1.noarch.rpm
+libsepol-2.9-5.cm1.aarch64.rpm
+libselinux-2.9-3.cm1.aarch64.rpm
+finger-0.17-4.cm1.aarch64.rpm
```

Comment:
do we know why finger is needed?

Reply:
`tcp_wrappers` needs it, which is in turn required by `audit` which is needed by `libsemanage` which is needed by `shadow-utils`. I'll comment below on the root cause of this chain.

Reply:
Actually, looking again the shadow-utils one is a BuildRequires... I can't find an obvious set of requirements here so I don't understand why tdnf is demanding it. Consider this an open question for now.

Reply:
Ok, looked into it. There is a dependency added by a generator for libsemanage, which kicks off this chain.